### ISSUE TITLE
Take your update check off of the main thread

### ISSUE THREAD

**Issue by RoboMWM** [ProjectKorra/ProjectKorra#934]
Yea 1) http stuff should _never_ be on the main thread and 2) boy do I love my startups delayed an extra 2 minutes.

```
[19:36:50] [Server thread/INFO]: [ProjectKorra] Enabling ProjectKorra v1.8.6
[19:38:31] [Server thread/INFO]: [ProjectKorra] Could not connect to ProjectKorra.com to check for updates
```

**Comment by RoboMWM** [comment]
Also am I blind or is there no way to disable this update check...

**Comment by StrangeOne101** [comment]
No, I don't think there is one. The update checker is very old, so it's probably time for a solid update on it.

**Comment by Hetag1216** [comment]
And so when will it be fixed? >.>

**Comment by Simplicitee** [comment]
When it's fixed

**Comment by jayoevans** [comment]
"Will be fixed in the next update." - Strange

**Comment by Hetag1216** [comment]
Just wanted to know if there is already an (un)official release date

**Comment by jayoevans** [comment]
As far as I'm aware this is (one of) the last task(s) remaining until release; however, we currently plan on having a public testing period before posting the update.

**Comment by Hetag1216** [comment]
Thank you, that's what i wanted to know.

**Comment by Varhagna** [comment]
Issue has been resolved but never closed, smh

Fixed by #937
